big_tests/rebar.config

```diff
@@ -31,22 +31,6 @@
   {pc, {git, "https://github.com/blt/port_compiler.git", {ref, "c2f3fb1"}}}
  ]}.
 
-{overrides,
-  %% Disable hex plugin for processone deps
-  %% All the deps are from corresponding rebar.configs
-  [{override, esip, [{plugins, []}, {deps, [
-        %% processone devs use Hex for their deps
-        %% Hex packages for these versions are missing
-        {stun, ".*", {git, "https://github.com/processone/stun", {tag, "1.0.16"}}},
-        {fast_tls, ".*", {git, "https://github.com/processone/fast_tls", {tag, "1.0.17"}}},
-        {p1_utils, ".*", {git, "https://github.com/processone/p1_utils", {tag, "1.0.10"}}}
-    ]}]},
-   {override, stun, [{plugins, []}, {deps, [
-        {fast_tls, ".*", {git, "https://github.com/processone/fast_tls", {tag, "1.0.17"}}},
-        {p1_utils, ".*", {git, "https://github.com/processone/p1_utils", {tag, "1.0.10"}}}
-    ]}]},
-   {override, fast_tls, [{plugins, [pc]}, {deps, [
-        {p1_utils, ".*", {git, "https://github.com/processone/p1_utils", {tag, "1.0.10"}}}
-    ]}]}
-
-  ]}.
+{overrides, [
+  {override, rebar3_codecov, [{deps, [{jiffy, "1.0.1"}]}]}
```

Comment:
What does it mean?

Reply:
Jiffy is used in this rebar plugin (rebar3_codecov) as well and is hardcoded there as {vsn,"0.15.2"}.
I could have left it as it is, as it is not used directly in release nor in tests, however just to not have any confusion regarding version I decided to upgrade it as well.
The only way to upgrade a dependency of dependency is to use override option in rebar.config.
I think it is an explicit way of telling in main project what we want. The other way would be to ask maintainer of codecov plugin to change it.

Reply:
it's ESL homemade plugging, update the version in the `rebar3_codecov` repo.
also let's not confuse anyone with extra override in our `rebar.config`

Reply:
Interestingly it is not in ESL repo. As are several of ESL made deps in MIM.

Reply:
we can clone it to ESL. ask Karol for help, he has required rights to create new ESL repo.